examples/private-registry.md

```diff
@@ -54,66 +54,66 @@ The following steps demonstrate how to enable and use the private Docker registr
 1. Prepare a DC/OS service definition:
 
     ```bash
-$ tee nginx-marathon.json <<-'EOF'
-{
-  "id": "/nginx",
-  "instances": 1,
-  "cpus": 0.5,
-  "mem": 128,
-  "container": {
-    "type": "DOCKER",
-    "docker": {
-      "image": "boot.dcos:5000/nginx",
-      "network": "HOST"
-    }
-  },
-  "healthChecks": [
+    $ tee nginx-marathon.json <<-'EOF'
     {
-      "protocol": "COMMAND",
-      "command": { "value": "service nginx status | grep -q 'nginx is running.'"},
-      "gracePeriodSeconds": 300,
-      "intervalSeconds": 60,
-      "timeoutSeconds": 20,
-      "maxConsecutiveFailures": 3
+      "id": "/nginx",
+      "instances": 1,
+      "cpus": 0.5,
+      "mem": 128,
+      "container": {
+        "type": "DOCKER",
+        "docker": {
+          "image": "boot.dcos:5000/nginx",
+          "network": "HOST"
+        }
+      },
+      "healthChecks": [
+        {
+          "protocol": "COMMAND",
+          "command": { "value": "service nginx status | grep -q 'nginx is running.'"},
+          "gracePeriodSeconds": 300,
+          "intervalSeconds": 60,
+          "timeoutSeconds": 20,
+          "maxConsecutiveFailures": 3
+        }
+      ],
+      "labels": {
+        "DCOS_SERVICE_NAME": "nginx",
+        "DCOS_SERVICE_SCHEME": "http",
+        "DCOS_SERVICE_PORT_INDEX": "0"
+      }
     }
-  ],
-  "labels": {
-    "DCOS_SERVICE_NAME": "nginx",
-    "DCOS_SERVICE_SCHEME": "http",
-    "DCOS_SERVICE_PORT_INDEX": "0"
-  }
-}
-EOF
-    ```
-1. Create a DC/OS service:
-
-    ```bash
-    $ dcos marathon app add nginx-marathon.json
+    EOF
     ```
+    1. Create a DC/OS service:
```

Comment:
Why are these two bullets indented? They're not part of "Prepare a DC/OS service definition"